react/Identification.tsx

```diff
@@ -149,6 +149,8 @@ const Identification: React.FC = () => {
       </span>
 
       <Input
+        autoComplete="email "
```

Comment:
is the little space in the end intentional?

```suggestion
        autoComplete="email"
```

Reply:
Nice catch! Removed this unnecessary whitespace and pushed the change